.changeset/slow-lemons-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+You can now see all options to select in product variant sheet when selecting attribute value from select
```

Comment:
The bug was about that already selected value filtered the list:

_The selected value is no longer filtering the options when using any select list in product variant table_